backend/plugins/frontline_api/src/modules/integrations/facebook/fieldUtils.ts

```
@@ -0,0 +1,76 @@
+  let fields: Array<{
+    _id: number;
+    name: string;
+    group?: string;
+    label?: string;
+    type?: string;
+    validation?: string;
+    options?: string[];
+    selectOptions?: Array<{ label: string; value: string }>;
+  }> = [];
+
+  let schema = schemas[collectionType].schema;
```

ESLint

[error] 'schema' is never reassigned. Use 'const' instead.

(prefer-const)